kmip/pie/client.py

```diff
@@ -541,6 +542,59 @@ def activate(self, uid=None):
             message = result.result_message.value
             raise exceptions.KmipOperationFailure(status, reason, message)
 
+    def revoke(self, code, uid=None, message=None,
```

Comment:
Let's rename `code` to `reason` and let's drop `message=None` entirely. There's no need for the message, since the reason enumeration "is" the message.